python-checks/src/main/java/org/sonar/python/checks/IdentityComparisonWithCachedTypesCheck.java

```diff
@@ -44,6 +46,8 @@
 public class IdentityComparisonWithCachedTypesCheck extends PythonSubscriptionCheck {
   private static final String MESSAGE_IS = "Replace this \"is\" operator with \"==\"; identity operator is not reliable here.";
   private static final String MESSAGE_IS_NOT = "Replace this \"is not\" operator with \"!=\"; identity operator is not reliable here.";
+  public static final String IS_QUICK_FIX_MESSAGE = "Replace with \"==\".";
+  public static final String IS_NOT_QUICK_FIX_MESSAGE = "Replace with \"!=\".";
```

Comment:
Can you remove the full stop at the end of the sentence?

I forgot it was actually written and was going a bit by instinct, but [here](https://github.com/SonarSource/sonarlint-core/blob/master/plugin-api/src/main/java/org/sonarsource/sonarlint/plugin/api/issue/NewQuickFix.java#L46) there are some small guidelines from SonarLint for how to phrase quick fix messages.

Reply:
done